test/parallel/test-next-tick-errors.js

```diff
@@ -20,6 +20,28 @@ process.nextTick(function() {
   order.push('C');
 });
 
+assert.throws(
```

Comment:
This should be removed because having just `process.nextTick` (no parens/uninvoked) is never going to throw, it will always return the function itself.